PSix/EventsListViewController.swift

```diff
@@ -7,15 +7,53 @@
 //
 
 import UIKit
+import Parse
 
-class EventsListViewController: UIViewController {
+class EventsListViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    static let EVENT_CELL_ID = "EventCell"
+    
+    private var userCreatedEvents = Events()
+    
+    @IBOutlet weak var eventsListTable: UITableView!
     
     override func viewWillAppear(animated: Bool) {
-        if !User.isLoggedIn() {
+        if !ParseUserSession.isLoggedIn {
             let onboardingStoryboard = UIStoryboard(name: "Onboarding", bundle: nil)
             let onboardingVC = onboardingStoryboard.instantiateViewControllerWithIdentifier("OnboardingViewController") as! UIViewController
             presentViewController(onboardingVC, animated: true, completion: nil)
+        } else if let currentUser = ParseUserSession.currentUser {
+            FacebookService.getFutureEventsCreatedByUser(currentUser, failure: nil) { [unowned self] (eventsData) -> Void in
+                self.extractEventsFromFbResponse(eventsData)
+                self.updateUI()
+            }
         }
+        
+        eventsListTable.dataSource = self
+        eventsListTable.delegate = self
+    }
+    
+    func updateUI() {
+        eventsListTable.reloadData()
+    }
+    
+    func extractEventsFromFbResponse(eventsData: NSArray) {
+        let eventsParser = FacebookEventsParser(eventsData: eventsData)
+        userCreatedEvents += eventsParser.events
```

Comment:
you still have the issue here that you never reset userCreatedEvents between refreshes